CLI/src/main/java/main/Drifty_CLI.java

```diff
@@ -53,20 +58,44 @@ public static void main(String[] args) {
         printBanner();
         if (args.length > 0) {
             link = null;
-            String name = null;
-            String location = null;
+            yamlFilePath = Paths.get(Program.get(Program.DRIFTY_PATH), YAML_FILENAME).toAbsolutePath().toString();
             for (int i = 0; i < args.length; i++) {
                 switch (args[i]) {
-                    case HELP_FLAG, HELP_FLAG_SHORT -> {
-                        help();
-                        System.exit(0);
+                    case HELP_FLAG, HELP_FLAG_SHORT -> help();
+                    case ADD_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgBatchError("No URL provided.");
+                            System.exit(1);
+                        }
+                        for (int j = i + 1; j < args.length; j++) {
+                            addUrlToFile(args[j]);
+                        }
                     }
-                    case NAME_FLAG, NAME_FLAG_SHORT -> name = args[i + 1];
-                    case LOCATION_FLAG, LOCATION_FLAG_SHORT -> location = args[i + 1];
-                    case VERSION_FLAG, VERSION_FLAG_SHORT -> {
-                        printVersion();
-                        System.exit(0);
+                    case LIST_FLAG -> listUrls();
+                    case REMOVE_FLAG -> {
+                        if (i + 1 >= args.length) {
+                            messageBroker.msgBatchError("No line number provided for removal.");
+                            System.exit(1);
+                        }
+                        for (int j = i + 1; j < args.length; j++) {
+                            if ("all".equalsIgnoreCase(args[j])) {
+                                removeAllUrls();
+                            } else {
+                                removeUrl(args[j]);
+                            }
+                        }
                     }
+                    case GET_FLAG -> {
+                        batchDownloading = true;
+                        // Logic to download URLs goes here
+                        System.out.println("Downloading URLs...");
```

Comment:
@Femi-lawal I think there's no need to print this line because it is automatically handled by the `batchDownloader()` method.